_1327/documents/admin.py

```diff
@@ -3,21 +3,15 @@
 from polymorphic.admin import PolymorphicParentModelAdmin
 from reversion.admin import VersionAdmin
 
-from _1327.information_pages.admin import InformationDocumentAdmin
 from _1327.information_pages.models import InformationDocument
-from _1327.minutes.admin import MinutesDocumentAdmin
 from _1327.minutes.models import MinutesDocument
 
 from .models import Attachment, Document
 
 
 class DocumentAdmin(GuardedModelAdmin, VersionAdmin, PolymorphicParentModelAdmin):
 	base_model = Document
-	child_models = (
-		(InformationDocument, InformationDocumentAdmin),
-		(MinutesDocument, MinutesDocumentAdmin),
-	)
-
+	child_models = (InformationDocument, MinutesDocument)  # no Polls in here?
```

Comment:
that is correct. polls should not be edited on the admin pages (actually, everything else will at some point also not be accessible from the admin pages anymore)